src/components/order-service/order-service.html

```diff
@@ -75,7 +75,7 @@
         type="button"
         class="btn btn-primary wizard-pf-next"
         ng-if="$ctrl.currentStep.id === 'configure'"
-        ng-disabled="!$ctrl.selectedProject || $ctrl.currentStep.id === 'results'"
+        ng-disabled="!$ctrl.selectedProject || $ctrl.currentStep.id === 'results' || $ctrl.invalidNewProject()"
```

Comment:
Hi @spadgett, the new project form is in the select project component, not in order service html.  Not sure what you are suggesting here.